index.html

```diff
@@ -37,7 +37,8 @@
                     <div class="inner"></div>
```

Comment:
@KatarzynaChmielecka I think it would be better to generate the wand dynamically, it shouldnt be in the index.html as it only appears while the game is on.